torax/geometry.py

```diff
@@ -28,6 +28,7 @@
 import jax.numpy as jnp
 import numpy as np
 import scipy
+from contourpy import contour_generator
```

Comment:
Can you add new dependencies to here

https://github.com/google-deepmind/torax/blob/main/pyproject.toml

Also can you make it such that the dependencies are ordered alphabetically - although keeping the groupings